<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{background-color:#fff9a2;}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Listado de Clientes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonAtras">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Atr√°s</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonAceptar">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Aceptar</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>631</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{background-color:#bbfa62;
border: 2px solid black;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>591</width>
       <height>381</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>587</width>
        <height>377</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;
&lt;ui version=&quot;4.0&quot;&gt;
 &lt;widget name=&quot;__qt_fake_top_level&quot;&gt;
  &lt;widget class=&quot;QScrollArea&quot; name=&quot;scrollArea&quot;&gt;
   &lt;property name=&quot;geometry&quot;&gt;
    &lt;rect&gt;
     &lt;x&gt;110&lt;/x&gt;
     &lt;y&gt;70&lt;/y&gt;
     &lt;width&gt;581&lt;/width&gt;
     &lt;height&gt;391&lt;/height&gt;
    &lt;/rect&gt;
   &lt;/property&gt;
   &lt;property name=&quot;widgetResizable&quot;&gt;
    &lt;bool&gt;true&lt;/bool&gt;
   &lt;/property&gt;
   &lt;widget class=&quot;QWidget&quot; name=&quot;scrollAreaWidgetContents&quot;&gt;
    &lt;property name=&quot;geometry&quot;&gt;
     &lt;rect&gt;
      &lt;x&gt;0&lt;/x&gt;
      &lt;y&gt;0&lt;/y&gt;
      &lt;width&gt;579&lt;/width&gt;
      &lt;height&gt;389&lt;/height&gt;
     &lt;/rect&gt;
    &lt;/property&gt;
    &lt;widget class=&quot;QListWidget&quot; name=&quot;listClientes&quot;&gt;
     &lt;property name=&quot;geometry&quot;&gt;
      &lt;rect&gt;
       &lt;x&gt;0&lt;/x&gt;
       &lt;y&gt;0&lt;/y&gt;
       &lt;width&gt;581&lt;/width&gt;
       &lt;height&gt;391&lt;/height&gt;
      &lt;/rect&gt;
     &lt;/property&gt;
    &lt;/widget&gt;
   &lt;/widget&gt;
  &lt;/widget&gt;
 &lt;/widget&gt;
 &lt;resources/&gt;
&lt;/ui&gt;
</string>
      </property>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>591</width>
         <height>381</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QListWidget{background-color:white;}</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonAceptar</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>lanzarDetalleCliente()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>638</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>605</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAtras</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>lanzarPrincipal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>512</y>
    </hint>
    <hint type="destinationlabel">
     <x>-3</x>
     <y>501</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>lanzarDetalleCliente()</slot>
  <slot>lanzarPrincipal()</slot>
 </slots>
</ui>
